app/views/courses/show.html.erb

```diff
@@ -61,6 +63,7 @@
       <%= render partial: 'courses/advice' %>
 
       <%= render Courses::ApplyComponent.new(course) %>
```

Comment:
Looks like you forgot to remove the component at the bottom, so I think there are now two on the page. This should also fix your failing test 😄